.travis.yml

```diff
@@ -87,15 +88,9 @@ install:
    - cd $TRAVIS_BUILD_DIR
    - conda env create -f tardis_env27.yml
    - source activate tardis
-   #trouble with building due to segfault at cython (https://github.com/cython/cython/issues/2199)
-   #remove if we can get normal cython through conda
```

Comment:
Should we leave this in as a comment? This way, we'll know what to do when some strange cython error appears in the future.

Reply:
That’s why I put it in

Reply:
I was thinking of keeping the procedure of getting bleeding-edge cython via github and compiling it with custom flags as a comment block and providing some additional information.